cognite_toolkit/_cdf_tk/loaders/_resource_loaders/transformation_loaders.py

```diff
@@ -194,23 +194,6 @@ def _are_equal(
 
         return self._return_are_equal(local_dumped, cdf_dumped, return_dumped)
 
-    @staticmethod
-    def _get_query_file(filepath: Path, transformation_external_id: str | None) -> Path | None:
```

Comment:
Cleanup that could have been done before, as we now know the query should hold a path to the sql file